fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/codec/LuceneOptimizedPostingsFormat.java

```diff
@@ -44,7 +44,7 @@
 @AutoService(PostingsFormat.class)
 public class LuceneOptimizedPostingsFormat extends PostingsFormat {
     PostingsFormat postingsFormat;
-    private static boolean allowCheckDataIntegrity = true;
+    static boolean allowCheckDataIntegrity = true;
```

Comment:
If this sticks around for any amount of time, it would probably be good to move it to some sort of `LuceneGlobalConfig` class or something like that.